
class ValidationException(Exception):
    pass


class RepromptException(Exception):
    pass
